app/scripts/proactive/view/dom.js

```diff
@@ -437,6 +437,19 @@ define(
             }
         }
 
+        function getToggledTasks() {
+            var toggler = document.getElementsByClassName("caretUL");
+            var i;
+            var toggledTasks = [];
+            for (i = 0; i < toggler.length; i++) {
```

Comment:
`i` could be declared in the loop like `for (let i = 0; i < cars.length; i++) {`

Reply:
ok, thanks for the trick :-)